hack/.linted_packages

```diff
@@ -388,7 +388,6 @@ staging/src/k8s.io/kube-aggregator/pkg/client/listers/apiregistration/v1alpha1
 staging/src/k8s.io/sample-apiserver
```

Comment:
I don't follow.  All I'm doing here is disabling linting for test/e2e_federation/upgrade so it's possible to dot import ginkgo for the upgrade test.